test.sh

```diff
@@ -9,7 +9,7 @@
 # Execute in docker-compose.yml directory, it will create containers and
 # test them.
 
-cat >>docker-compose.override.yml <<EOT
+[ ! -f docker-compose.override.yml ] && cat >>docker-compose.override.yml <<EOT
```

Comment:
Maybe it would be better to start with fresh override every time?

```suggestion
cat > docker-compose.override.yml <<EOT
```

Reply:
# 💯

Appreciate this :)